pywps/inout/basic.py

```diff
@@ -297,7 +297,9 @@ def file(self, value):
     def data(self):
         """Read file and return content."""
         if self._data is None:
-            with open(self.file, mode=self._openmode()) as fh:
+            openmode = self._openmode()
+            kwargs = {} if 'b' in openmode else {'encoding': 'utf8'}
```

Comment:
`encoding` is only available in Python 3.x? We still have Python 2.7 compatibility. Using `io.open` might work for 2.7 and 3.x.